eis_qgis_plugin/eis_wizard/mineral_proxies/workflows/interpolate.py

```diff
@@ -52,6 +52,7 @@ def __init__(self,
         self.interpolation_method: QComboBox
         self.interpolation_method_pages: QComboBox
         self.power: QgsDoubleSpinBox
+        self.search_radius: QgsDoubleSpinBox
```

Comment:
The max value seemed to be 99 when I tested this in Wizard. Could set maxValue something like 99999, should be enough